.github/workflows/misc-migtests.yml

```diff
@@ -43,6 +43,12 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
 
+      - name: "Enable postgres with pg_stat_statements loaded via shared_preload_libraries"
+        run: |
+          docker exec ${{ job.services.postgres.id }} sh -c "echo \"shared_preload_libraries = 'pg_stat_statements'\" >> /var/lib/postgresql/data/postgresql.conf"
+          docker restart ${{ job.services.postgres.id }}
+          sleep 10
```

Comment:
Just curious, what are the scenarios when user has to add pgss in `shared_preload_libraries`
Or is it part of the initial setup for the extension? Because i dont remember doing it on my machine manually.

Reply:
Yes seems like its required by the initial setup itself.
From PG docs
`The module must be loaded by adding pg_stat_statements to [shared_preload_libraries](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-SHARED-PRELOAD-LIBRARIES) in postgresql.conf, because it requires additional shared memory.`